packages/extension-chakra-storefront/src/theme/components/project/_app.js

```diff
@@ -4,10 +4,11 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import {theme} from '@chakra-ui/theme'
```

Comment:
can't import theme object in Chakra V3, Let's use a hardcode value for now. Will circle back when we start working on fixing component styling